web_export_view/static/src/js/web_export_view.js

```diff
@@ -85,7 +85,8 @@ openerp.web_export_view = function (instance) {
                                 }
                             }
                             else if (cell.classList.contains("oe_list_field_integer")) {
-                                export_row.push(parseInt(text));
+                                tmp2 = text.replace(instance.web._t.database.parameters.thousands_sep, "");
```

Comment:
all the js code in this file is a mess (using global scope for all variables), so this makes it only a bit worse i think. but a PR that fixes all global variables would be a good thing of course